File: laba3/lab3.cs
using System;

class Program {
    static int SearchEven(int num) {
        int count = 0;
        while (num > 0) {
            int last = num % 10;
            if (last % 2 == 0) count++;
            num /= 10;
        }
        return count;
    }

    static void Main() {
        Console.Write("Enter number of numbers: ");
        int n = int.Parse(Console.ReadLine());
        for (int i = 0; i < n; i++) {
            Console.Write("Enter number: ");
            int k = int.Parse(Console.ReadLine());
            Console.WriteLine(SearchEven(k));
        }
    }
}

File: laba3/lab3.rs
use std::io;

fn search_even(mut num: i32) -> i32 {
    let mut count = 0;
    while num > 0 {
        let last = num % 10;
        if last % 2 == 0 {
            count += 1;
        }
        num /= 10;
    }
    count
}

fn main() {
    let mut input = String::new();
    println!("Enter number of numbers: ");
    io::stdin().read_line(&mut input).unwrap();
    let n: i32 = input.trim().parse().unwrap();
    
    for _ in 0..n {
        input.clear();
        println!("Enter number: ");
        io::stdin().read_line(&mut input).unwrap();
        let k: i32 = input.trim().parse().unwrap();
        println!("{}", search_even(k));
    }
}
    